File: app/src/main/java/com/example/movieslisttask/service/APIService.java
package com.example.movieslisttask.service;

import com.example.movieslisttask.model.MoviesResponse;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

/**
 * Created by alexkorolov on 2020-01-15.
 */
public interface APIService {

    @GET("movie/popular")
    Call<MoviesResponse> getPopularMovies(@Query("api_key") String apiKey);
}

File: app/src/main/java/com/example/movieslisttask/service/APIService.kt
package com.example.movieslisttask.service

import com.example.movieslisttask.service.responses.MoviesResponse
import com.example.movieslisttask.tools.Consts
import retrofit2.Response
import retrofit2.http.GET

/**
 * Created by alexkorolov on 2020-01-15.
 */
interface APIService {

    @GET("movie/popular?api_key=${Consts.MOVIES_API_KEY}")
    suspend fun getPopularMovies(): Response<MoviesResponse>

}